modules/linear_boltzmann_solvers/executors/pi_keigen.cc

```diff
@@ -172,11 +172,14 @@ PowerIterationKEigen::SetLBSFissionSource(const std::vector<double>& input, cons
   if (not additive)
     Set(q_moments_local_, 0.0);
 
-  active_set_source_function_(front_gs_,
```

Comment:
I don't see a reason for any of the members named `front_<...>`. Should we just get rid of them?

Reply:
`front_wgs_context` is used to print the number of sweeps (which I'm not sure we even need to do). For this type of problem, I'm not sure it's a interesting statistic.

Reply:
Either way, that could be done without `front_wgs_context` as a member though. Also, shouldn't we sum the number of sweeps in each groupset instead? Why would the first one contain information cumulative for all of them?

Reply:
Yes, it should be the sum. I'm going to remove that anyway... as I said, I don't think it's a useful stat here. However, `pi_keign_smm` and `pi_keigen_scdsa` depend on those member variables, so they can't be removed. I think `smm` checks for a single groupset, but I don't know about `scdsa`. If it doesn't, I'll add that.

Reply:
Awesome. Yep, the others do (somewhat unnecessarily) because they are only implemented for a single groupset. That's a future issue to tackle though.

Reply:
I hate to say this after you removed it... But I forgot that I used print statement about sweeps that was removed in my SMM paper. Is there any way that could be added back in to print the sum of sweep counts over groupsets? It was useful to have to compare how many fewer sweeps you need using acceleration schemes. Sorry I did not think of that before

Reply:
No problem. Fixed.